test/__mocks__/text.mock.ts

```diff
@@ -184,3 +184,4 @@ export const BENEFICIAL_OWNER_MANAGING_OFFICER_TYPE_LEGEND_TEXT = "Which type of
 export const UPDATE_BENEFICIAL_OWNER_TYPE_PAGE_REDIRECT = "Found. Redirecting to /update-an-overseas-entity/update-beneficial-owner-type";
 export const UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS = "Contact details";
 export const UPDATE_CHECK_YOUR_ANSWERS_WHO_CAN_WE_CONTACT = "Who can we contact about this application?";
+export const UPDATE_REGISTRABLE_BENEFICIAL_OWNER_TITLE = "Has anyone become or ceased to be a registrable beneficial owner during the update period?";
```

Comment:
Same comment as above

Reply:
This one is fine. Is just a check for the title